### ISSUE TITLE
[Bug]: Bottles Hangs on get_processes

### ISSUE THREAD

**Issue by Alzarath** [bottlesdevs/Bottles#1532]
### Describe the bug

Every time `get_processes` is called (determined by watching terminal output and seeing `(INFO) Using Wine debug tool -- get_processes` printed during the hangs), Bottles hangs for about half a second.

In my experience, it is called when a Bottle's section is opened, when the Programs list is updated via adding or removing a Program from the list, and most noticeably, when opening *and* closing the "Change launch options" dialog for a Program. It appears to call this twice for each Program in the Bottle's list, meaning for each Program, the bottle will hang for about a second.

My heaviest Bottle has 7 Programs, so it hangs for 7 seconds every time one of the above triggers. This can add up and cause frustration.

### To Reproduce

Open a Bottle from the main window.

### Package

AUR ("bottles" package)

### Distribution

Arch Linux

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/share/vulkan/icd.d/nvidia_icd.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.9-arch1-1
Distro:
    Name: Arch Linux
    Version: Unknown
Disk:
    Total: 527295578112
    Free: 424525901824
RAM:
    MemTotal: 31.3GiB
    MemAvailable: 23.2GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.35'
Bottles_envs: null
```

### Display Protocol

X11

### Display Backend used by Bottles

X11

### Additional context

I use `bspwm` if that happens to matter.

**Comment by mirkobrombin** [comment]
That function is async, what hangs is the wineserver turning in. We are working to improve this and make the wineserver start async too

**Comment by mirkobrombin** [comment]
Fixed in 2022.6.14
